src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -444,7 +442,11 @@ private static boolean isAbstract(final CtMethod m) {
   }
 
   /**
-   * Checks whether the <code>ABSTRACT</code> flag of a class is set.
+   * Checks whether the <code>ABSTRACT</code> flag of a class or any of its
+   * methods is set.
+   * TODO Check if looking at the flags of its methods is really necessary.
```

Comment:
This is indeed the case, see https://docs.oracle.com/javase/tutorial/java/IandI/abstract.html:
> If a class includes abstract methods, then the class itself must be declared abstract

Instead of adding a TODO I suggest fixing this. TODOs tend to be overlooked for a long time.

Reply:
Sounds fair. In that case I would mention it in the TODO so that it's clear the fix is not easy for whatever reason and create a ticket for it perhaps. But I would be surprised if something breaks (happens fairly often to me though in our codebase :wink: ).

Reply:
@svorenova I implemented the change in a new commit "Remove redundant checks for abstract methods", which replaces the previous commit about documentation. Please re-review - I will wait for CI on the pointer bump to pass before I merge this of course.